### PR TITLE
Register warden Mwita

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden Mwita

@MwitahJob

### CODE REVIEW

**Review by MwitahJob — COMMENTED** [review]
Okay

**Review by MwitahJob — APPROVED** [review]

**Review by MwitahJob — CHANGES_REQUESTED** [review]
Merge.

File: _data/handles/Mwita.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "handle": "Mwita",
+  "link": "https://twitter.com/MwitahJob",
+  "moralisId": "CPB4MMUvt7B3tweVMaNGo3yO",
+  "image": "./avatars/Mwita.jpeg"
+}
```
Comment (MwitahJob) [inline _data/handles/Mwita.json:6]: Make changes.

**Review by MwitahJob — APPROVED** [review]
Make changes.

**Comment by SamusElderg** [comment]
Have contacted user to change their nickname to match application

**Review by SamusElderg — APPROVED** [review]

**Pull request merged**